tests/integration/helpers/factories/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is required in order to be able to do:
`from factories.db.invenio_oauthclient import TestRemoteToken`
The same for the other init files.